### PR TITLE
DROOLS-2263 : Unexpected results in GDST when using enumerations with commas

### PR DESCRIPTION
**Pull request by Rikkola**
https://issues.jboss.org/browse/DROOLS-2263

(cherry picked from commit 69ca5dbbea8381dd9dedba877e52bd395b21a025)

Bundle:
https://github.com/kiegroup/kie-soup/pull/29
https://github.com/kiegroup/drools/pull/1727
https://github.com/kiegroup/drools-wb/pull/777

### CODE REVIEW

**Review by manstis — APPROVED** [review]

File: drools-workbench-models/drools-workbench-models-commons/src/main/java/org/drools/workbench/models/commons/backend/rule/RuleModelDRLPersistenceImpl.java [hunk 1]
```diff
@@ -1314,17 +1315,11 @@ protected void populateValueList(final StringBuilder buf,
                 workingValue = workingValue.substring(0,
                                                       workingValue.length() - 1);
             }
-            final String[] values = workingValue.split(",");
+
+            final String[] values = ListSplitter.split("\"", true, workingValue);
```
Comment (jomarko) [inline drools-workbench-models/drools-workbench-models-commons/src/main/java/org/drools/workbench/models/commons/backend/rule/RuleModelDRLPersistenceImpl.java:1319]: Please add test similar to:
`org.drools.workbench.models.commons.backend.rule.RuleModelDRLPersistenceTest#testInOperatorString`

File: drools-workbench-models/drools-workbench-models-guided-dtable/src/main/java/org/drools/workbench/models/guided/dtable/backend/GuidedDTDRLPersistence.java [hunk 2]
```diff
@@ -742,25 +745,30 @@ private void doCondition(List<BaseColumn> allColumns,
     /**
      * take a CSV list and turn it into DRL syntax
      */
-    String makeInList(String cell) {
+    String makeInList(final String cell) {
```
Comment (jomarko) [inline drools-workbench-models/drools-workbench-models-guided-dtable/src/main/java/org/drools/workbench/models/guided/dtable/backend/GuidedDTDRLPersistence.java:748]: Please add test similar to:
`org.drools.workbench.models.guided.dtable.backend.GuidedDTDRLPersistenceTest#testLHSInOperator`

**Review by jomarko — COMMENTED** [review]
@Rikkola Thanks, may I ask you for increasing coverage a little bit?

**Review by jomarko — APPROVED** [review]

**Comment by manstis** [comment]
Jenkins please retest this.

**Comment by jomarko** [comment]
@manstis this is ready for merge.

Ensemble 1/4.

**Pull request merged**